### DISCUSSION TITLE
Unexpected git hash commit in Nuget package version

### DISCUSSION THREAD

**Discussion by Woudjee** [NuGet/Home#13608] [General]
Hi,

I have been experimenting with the automatic creation of Nuget packages in my Azure DevOps pipeline and have been running into an issue where the git commit hash is added to the version of my Nuget package. For me, this is unexpected behaviour. I have spent a lot of time trying to understand how to change this. After not being succesful, I postponed working on this.

Today I continued and pushed my first .NET Core Nuget package (as well as a .NET Framework package). I guess the screenshot below explains well which one is which. It really drew my attention that the .Core Nuget does not have the git commit hash.

![image](https://github.com/NuGet/Home/assets/84737524/edbd244f-6d6b-4d32-89e1-df243a816bd6)

Both projects use the SDK .csproj setup and are identical except for some package references. See below:

![image](https://github.com/NuGet/Home/assets/84737524/7bb72e12-7f8e-4bf7-b85e-0c116ba2fa03)

**Question: how do I remove the git commit hash from my Nuget version for my .NET Framework (or .NET Standard) Nuget package?**

Thank you for your suggestions!

Woudjee

**Comment by baronfel** [comment]
See the breaking change documentation for .NET 8 [here](https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/source-link) to learn more about this change and how to change it.

**Comment by Woudjee** [comment]
Omg it had to be something as simple as this. I will try it out straight away!

**Comment by Woudjee** [comment]
Btw, this works as expected now! :D
